src/Cryptography/Algorithms/AlgorithmMd5.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include "IHashFunction.h"
+
+#include <memory>
+
+namespace cryptography
+{
+    std::unique_ptr<IHashFunction> CreateMd5();
+}
```

Comment:
Fatal  missing `// crytopgraphy `, clang-format missing. entire PR  invalidated
![image](https://github.com/user-attachments/assets/7e04a791-c8c0-4aff-a1a6-5390313ce5a6)

Reply:
All cpp files are clang-format checked and my IDE auto formats on save. So if there is no comment then there shouldnt be :P

See ShortNamespaceLines on https://clang.llvm.org/docs/ClangFormatStyleOptions.html